tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/IClusterNodeIntegrationBis.cs

```diff
@@ -0,0 +1,61 @@
+// <copyright file="IClusterNodeIntegrationBis.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Couchbase
+{
+    /// <summary>
+    /// Couchbase IclusterNode 3.2 calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+       AssemblyName = CouchbaseCommon.CouchbaseClientAssemblyName,
+       TypeName = "Couchbase.Core.ClusterNode",
+       MethodName = "ExecuteOp",
+       ReturnTypeName = ClrNames.Task,
+       ParameterTypeNames = new[] { CouchbaseCommon.CouchbaseConnectionV3TypeName, CouchbaseCommon.CouchbaseOperationV3TypeName, "Couchbase.Core.IO.Operations.CancellationTokenPair" },
+       MinimumVersion = "3.1.3",
+       MaximumVersion = CouchbaseCommon.MaxVersion3,
+       IntegrationName = CouchbaseCommon.IntegrationName)]
+    // ReSharper disable once InconsistentNaming
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class IClusterNodeIntegrationBis
```

Comment:
I had to look up what "bis" and "ter" mean 😂

Reply:
I haven't found better names to be honest. Not proud of this one :)